mlir/include/mlir/Dialect/LLVMIR/LLVMAttrDefs.td

```diff
@@ -891,6 +891,8 @@ def LLVM_AliasScopeAttr : LLVM_Attr<"AliasScope", "alias_scope"> {
     }
     ```
 
+    The first attribute can either be a DistinctAttr or a StringAttr.
```

Comment:
Should we verify this to ensure the export does not explode? I do not recall what our policy on verifying metedata attributes is, tbh.

Reply:
Having a verifier would be nice but we have very little verification right now. So let's say this is optional.